### PR TITLE
Added animation

### PR DESCRIPTION
**Pull request by dalvishruti14**

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -1766,6 +1766,13 @@ let cards = [
     author: 'Gaurav Saxena',
     githubLink: 'https://github.com/Gauravsaxena816'
   },
+  {
+    artName: 'Rocket Launch',
+    pageLink: './Art/dalvishruti14/index.html',
+    imageLink: './Art/dalvishruti14/rocketlaunch.gif',
```
Comment (sophiabrandt) [inline include.js:1772]: Have you checked if this works?

Your file name is `RocketLaunch.gif` (camel case), but in your entry you wrote `rocketlaunch.gif` (all lower case).

Unix systems (and the pipeline runs on Ubuntu) distinguish between cases.

So I suggest that you double-check or adjust your link to the `.gif`.

**Comment by LaurelineP** [comment]
#2167 was merged already - please avoid MR duplications

**Comment by LaurelineP** [comment]
https://github.com/zero-to-mastery/Animation-Nation/pull/2167 is a duplicated MR - which was already merged:
please avoid MR duplications
This is why this would be closed.
Thanks for your understanding

**Comment by LaurelineP** [comment]
https://github.com/zero-to-mastery/Animation-Nation/pull/2167 is a duplicated MR - which was already merged:
please avoid MR duplications
This is why this would be closed.
Thanks for your understanding

**Pull request closed without merging**